.github/workflows/build.yml

```diff
@@ -0,0 +1,185 @@
+          fi
+
+          echo "THE_ENV=uat" >> $GITHUB_ENV
+          echo "TARGET_ENV_UPPERCASE=UAT" >> $GITHUB_ENV
+
+      ## Set Release variables
+      - name: Release
+        if: ${{ startsWith(github.ref, 'refs/heads/main') }}
+        run: |
+            echo "THE_ENV=ops" >> $GITHUB_ENV
+            echo "TARGET_ENV_UPPERCASE=OPS" >> $GITHUB_ENV
+            echo "Modifying version number ${{ env.project_version}}"
+            THE_VERSION=${{ env.project_version }}
+            echo "THE_VERSION=${THE_VERSION//-*}" >> $GITHUB_ENV
+
+
+      - name: Run Snyk as a blocking step
+        uses: snyk/actions/node@master
+        env:
+            SNYK_TOKEN: ${{ secrets.SNYK_TOKEN }}
+        with:
+          command: test
+          args: >
+              --org=${{ secrets.SNYK_ORG_ID }}
+              --project-name=${{ github.repository }}
+              --severity-threshold=high
+              --fail-on=all
+
+
+      - name: Run Snyk on Node
+        uses: snyk/actions/node@master
+        env:
+          SNYK_TOKEN: ${{ secrets.SNYK_TOKEN }}
+        with:
+          command: monitor
+          args: >
+            --org=${{ secrets.SNYK_ORG_ID }}
+            --project-name=${{ github.repository }}
+
+
+      - name: Commit Version Bump
+        # If building develop, a release branch, or main then we commit the version bump back to the repo
+        if: |
+          github.ref == 'refs/heads/develop' ||
+          github.ref == 'refs/heads/main'    ||
+          startsWith(github.ref, 'refs/heads/release')
+        run: |
+          git config --global user.name 'hitide-ui bot'
+          git config --global user.email 'hitide-ui@noreply.github.com'
+          git commit -am "/version ${{ env.THE_VERSION }}"
+          git push
+
+      - name: Push Tag
+        if: |
+          github.ref == 'refs/heads/develop' ||
+          github.ref == 'refs/heads/main'    ||
+          startsWith(github.ref, 'refs/heads/release')
+        run: |
+          git config user.name "${GITHUB_ACTOR}"
+          git config user.email "${GITHUB_ACTOR}@users.noreply.github.com"
+          git tag -a "${{ env.THE_VERSION }}" -m "Version ${{ env.THE_VERSION }}"
+          git push origin "${{ env.THE_VERSION }}"
+
+      # Setup Node to install and test
+      - name: Setup Node
+        uses: actions/setup-node@v3
+        with:
+          node-version: 14
+
+      - name: NPM install & NPM Lint
+        run: |
+          npm install
+          npm run build
+
+      - name: Make Config Files
+        run: |
+          cat ./configs/hitideConfig-${{ env.THE_ENV }}.js > ./dist/hitideConfig.js
+      
+      ## echo '{ "earthDataAppClientId": "'"${{ secrets[format('EARTH_DATA_LOGIN_CLIENT_ID_{0}', env.TARGET_ENV_UPPERCASE)] }}"'",  "googleTagManagerId": "'"${{ secrets[format('GOOGLE_TAG_MANAGER_ID_{0}', env.TARGET_ENV_UPPERCASE)] }}"'"}' > ./dist/env.json
+
+      ## Deployment
+      - name: Set AWS Env Variables
```

Comment:
Do we need this and the "Configure AWS credentials" step?

Reply:
i guess not i saw Jonathan did both step so i can remove this, it should work with one setup